contrib/inventory/consul_io.py

```diff
@@ -133,7 +133,11 @@
 import re
 import argparse
 import sys
-import ConfigParser
+
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
```

Comment:
Are you asking that I refactor the entire inventory script? To be clear I was following the conventions of the existing maintainer to fix a bug that breaks the file.

As six is not not in the core projects `requirements.txt` for the ansible project I am concerned about breaking installs, while it is included in `ansible.module_utils` it is not included anywhere in `contrib/*` and will result in an unmet dependency.

Can you clarify the policy on contrib Python imports?

I am willing to rewrite but also am attempting to avoid unmet dependencies while also following `CODING_GUIDELINES.md`

Thanks